code/modules/mob/living/emote.dm

```diff
@@ -478,3 +478,14 @@
 				else
 					R.unbuckle_all_mobs()
 	..()
+
+/datum/emote/living/circle
+	key = "circle"
+	key_third_person = "circles"
+	restraint_check = TRUE
+
+/datum/emote/living/circle/run_emote(mob/user, params)
+	. = ..()
+	var/obj/item/weapon/circlegame/N = new(user)

```

Comment:
Talked to leoz on IRC and the if(.) is not actually needed so please disregard this review